cinema/serializers.py

```diff
@@ -59,6 +69,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
+    tickets_available = serializers.IntegerField(read_only=True)
```

Comment:
The `tickets_available` field is defined as read-only, but there is no logic provided to calculate the number of available tickets. Consider implementing a method to calculate this value based on the total capacity of the cinema hall and the number of tickets already sold.